app/utils/Assets.scala

```diff
@@ -0,0 +1,30 @@
+package utils
+
+import java.net.URL
+import controllers.CachedAssets.hashedPaths
+import play.api.Play
+import org.apache.commons.io.IOUtils
+
+object Assets {
+
+  def inlineResource(path: String) = {
+    val url = AssetFinder("public/" + hashedPaths.getOrElse(path, path))
+    IOUtils.toString(url)
```

Comment:
Since when do we use apache commons? I cannot see it in the build.sbt dependencies...